src/ffpuppet/bootstrapper.py

```diff
@@ -18,39 +16,56 @@
 
 
 class Bootstrapper:  # pylint: disable=missing-docstring
-    BUF_SIZE = 4096  # receive buffer size
-    POLL_WAIT: float = 1  # duration of initial blocking socket operations
-    PORT_MAX = 0xFFFF  # bootstrap range
-    PORT_MIN = 0x3000  # bootstrap range
-    PORT_RETRIES = 100  # number of attempts to find an available port
```

Comment:
You could have kept this instead of hardcoding 50 in the `__init__` args.

Reply:
Agreed.